### PR TITLE
feature -> develop | Added User Setting 

### PR DESCRIPTION
**Pull request by wri-vikram-wri231**
Fixed Sidebar

### CODE REVIEW

File: Entities/User.php [hunk 1]
```diff
@@ -583,11 +588,64 @@ public static function beforeUserActionValidation($request)
...
+        }
+
+        $inputs = $request->all();
+        $inputs['email'] = trim($inputs['email']);
+
+        $rules = array(
+            'email' => 'required|max:150',
+        );
+        $messages = array(
+            'email.required' => 'The email or username field is required.',
```
Comment (themodernpk) [inline Entities/User.php:606]: @wri-vikram-wri231 all validation messages should come from language file and use `trans('vaahcms::messages.inactive_account');`

Comment (themodernpk) [reply Entities/User.php:606]: Apply this to everywhere it is needed

File: Http/Controllers/ExtendController.php [hunk 2]
```diff
@@ -68,13 +68,12 @@ public static function sidebarMenu()
             $list[0] = [
                 'link' => self::$link,
                 'icon' => 'tachometer-alt',
-                'label'=> 'Dashboard'
+                'label'=> 'Dashboard',
+                'data_wdio' => "data_wdio_test"
```
Comment (themodernpk) [inline Http/Controllers/ExtendController.php:72]: `data_wdio_test` should be `data_wdio_dashboard`

**Review by themodernpk — CHANGES_REQUESTED** [review]

File: Entities/User.php (hunk 1 above)
Comment (wri-vikram-wri231) [reply Entities/User.php:606]: okay Sir , I will check in every models and controllers

**Pull request merged**